requirements.txt

```diff
@@ -7,6 +7,7 @@ pointpats>=2.2.0
 segregation>=2.0.0
 spaghetti>=1.6.2
 mgwr>=2.1.2
+momepy>=0.5.0
```

Comment:
```suggestion
```
As above.

Reply:
Merging #1230 will incorporate those changes (i.e., these commits will be integrated there).